### PR TITLE
Tentative program documentation

### PR DESCRIPTION
**Pull request by jesselanger2**
Edited README file with basic program documentation. Can be edited more in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,104 @@
...
+`pip install key-switcheroo`
+
+
+## How to use
+
+Once the package is installed, commands can be called from the user's CLI for both the *publisher* and *retriever* using different optional arguments.
+
+For help with command-line arguments,
+
+`switcheroo --help` or `switcheroo -h`
```
Comment (thehcma) [inline README.md:53]: let's use the actual executables as the example here

File: README.md [hunk 2]
```diff
@@ -1,2 +1,104 @@
...
+
+### Publisher
+
+When using the *publisher* for creating and publishing new SSH keys, the user has a couple of different *optional* arguments, in addition to the *required* arguments.
+
+**Required Arguments:**
+1. `hostname` - host server
+2. `user` - username of the connecting client
+
+**Optional Arguments:**
```
Comment (thehcma) [inline README.md:63]: how are we handling the AWS authentication credentials? (don't we need that to be configured as well?)

**Review by thehcma — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ygins) [reply README.md:63]: The plan is to leave it up to the user executing the script to have AWS CLI configured, and if that is configured correctly all is fine

File: README.md (hunk 1 above)
Comment (ygins) [reply README.md:53]: executables were created after this, will update

**Pull request merged**